pkg/ocicni/ocicni.go

```diff
@@ -423,7 +425,7 @@ func (plugin *cniNetworkPlugin) SetUpPod(podNetwork PodNetwork) ([]cnitypes.Resu
 			ip = conf.IP
 		}
 
-		result, err := network.addToNetwork(plugin.cacheDir, podNetwork, ifName, ip)
+		result, err := network.addToNetwork(plugin.cacheDir, podNetwork, ifName, ip, plugin.podCidr)
```

Comment:
This puts the same podCidr in each network. Is this intentional?

Reply:
Just as I said above, podCidr is a global config owned by cni manager (like ocicni), but not a config owned by a specific cni network.

Reply:
> Just as I said above, podCidr is a global config owned by cni manager (like ocicni), but not a config owned by a specific cni network.

IMO it's actually not...

Reply:
I've got your point. :)